### PR TITLE
allow for non integer principal point coordinates

### PR DESCRIPTION
**Pull request by jsalotti**
Removed a constraint that forced image dimensions to be even in order to have integer principal point in camera intrinsic matrix.

### CODE REVIEW

File: aloscene/camera_calib.py [hunk 1]
```diff
@@ -64,10 +64,6 @@ def __new__(
             x[0][1] = skew if skew is not None else 0
 
             if principal_point is None and plane_size is not None:
-                if plane_size[0] % 2 != 0 or plane_size[1] % 2 != 0:
-                    err = "The principal points (center of the plane) can't be infer automaticly."
-                    err += " The latter must be given when creating the CameraIntrinsic tensor."
-                    raise Exception(err)
                 principal_point = (plane_size[0] / 2, plane_size[1] / 2)
```
Comment (Data-Iab) [inline aloscene/camera_calib.py:67]: Isn't it necessary for the principal point to be integer?

Comment (jsalotti) [reply aloscene/camera_calib.py:67]: I do not see where it is necessary. If somewhere in the code, maybe the check should be done only there. 
Currently, if you resize a `SpatialAugmentedTensor` with a `cam_intrinsic` labels, the principal points can already become float, and it seems to be working as expected.

Comment (anhtu293) [reply aloscene/camera_calib.py:67]: I think principal point should be in float so that it could work with real calibrated camera too

**Review by Data-Iab — APPROVED** [review]

File: aloscene/camera_calib.py (hunk 1 above)
Comment (thibo73800) [reply aloscene/camera_calib.py:67]: Make sense

**Review by thibo73800 — APPROVED** [review]

**Pull request merged**